extra/fuegogin/adaptor.go

```diff
@@ -0,0 +1,72 @@
+package fuegogin
+
+import (
+	"log/slog"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/go-fuego/fuego"
+)
+
+func GetGin(s *fuego.OpenAPI, e gin.IRouter, path string, handler gin.HandlerFunc, options ...func(*fuego.BaseRoute)) *fuego.Route[any, any] {
+	return handleGin(s, e, "GET", path, handler, options...)
```

Comment:
Here and everywhere, use http.MethodGet, http.MethodPost and not "GET", "POST"

Reply:
Done